Formula/granite.rb

```diff
@@ -0,0 +1,35 @@
+class Granite < Formula
+  desc "GTK+ extensions for elementary OS"
+  homepage "https://launchpad.net/granite"
+  url "https://launchpad.net/granite/0.3/0.3.1/+download/granite-0.3.1.tar.xz"
+  sha256 "8ec1d61f9aba75f1b3a745e721288b0dfb34cb11d1307be80cef7b0571c2dec6"
+
+  depends_on "gtk+3"
+  depends_on "libgee"
+  depends_on "cmake" => :build
+  depends_on "vala" => :build
+
+  # patch for https://bugs.launchpad.net/granite/+bug/1581531
```

Comment:
I opened a bug report in the issue tracker (linked In the comment), suggesting two possible solutions (for the same bug) and uploaded a branch with this patch, implementing one of the two. Knowing the team, it might take a few days for them to get back to me.